README.md

```diff
@@ -211,6 +211,14 @@ The plugin will read the descriptions and schema of each resource and data sourc
 
 Please refer to `examples` folder to perform CRUD operations with Tanzu Mission Control provider for various resources
 
+# Troubleshooting
+
+## Executions of a different version of the provider
+Terraform will always look for the latest version of the provided and will use it even if you have just built a previous version.
```

Comment:
nit: version of the provided -> version of the provider

Reply:
fixed